ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js

```diff
@@ -0,0 +1,339 @@
+import React, { useState, useEffect } from 'react';
+import { customAxios as axios } from '../../helpers/customAxios';
+import {
+  Container,
+  Heading,
+  Text,
+  Stack,
+  Button,
+  ListItem,
+  Flex,
+  Grid,
+  Spacer,
+  List,
+  Box,
+  Image,
+  HStack,
+  useToast,
+} from '@chakra-ui/react';
+import SermonNotesEditorModal from './SermonNotesEditorModal';
+import { MdOutlineVideoLibrary } from 'react-icons/md';
+import { BiDonateHeart } from 'react-icons/bi';
+import { IoPeopleOutline } from 'react-icons/io5';
+import { IoBookOutline } from 'react-icons/io5';
+import { FaRegCalendarAlt } from 'react-icons/fa';
+
+export default function AdminSermonNotesContainer(props) {
```

Comment:
you missed this ? We should change this to class component so it follows our codebase practices
```suggestion
const AdminSermonNotesContainer = (props) => {
```
don't forget to add export default at the bottom